app/views/tables/_form.html.erb

```diff
@@ -71,13 +71,17 @@
         <td><%= f.number_field :onsite_price %> </td>
       </tr>
 
-
       <tr>
         <td><%= f.label :core, 'CORE table' %></td>
         <td><%= f.check_box :core %></td>
       </tr>
 
       <tr>
+        <td><%= f.label :information, 'Additional information for players (such as using PFS characters, etc).' %></td>
+        <td><%= f.text_area :information %>
```

Comment:
Reword: "Additional information for players" or "Additional information for players (character requirements, etc)"